src/main/java/org/jetbrains/plugins/spotbugs/gui/tree/model/RootNode.java

```diff
@@ -37,7 +37,7 @@
 
 public class RootNode extends AbstractTreeNode<VisitableTreeNode> implements VisitableTreeNode {
 
-	private static final Icon EXPAND_ICON = new MaskIcon(ResourcesLoader.loadIcon("fb-logo.svg"), JBColor.BLACK);
+	private static final Icon EXPAND_ICON = new MaskIcon(ResourcesLoader.loadIcon("bluesheild.svg"));
```

Comment:
just to confirm if we are fetching the right svg is the asset also called '...sheild'?

Reply:
ah it's is called sheild... resolving